### PR TITLE
Fix the coloring of the meta description

### PR DESCRIPTION
**Pull request by atimmer**
It should be gray when no meta description has been set in the form.
This was previously broken in develop.

Fixes https://github.com/Yoast/wordpress-seo/issues/2996

### CODE REVIEW

File: js/snippetPreview.js [hunk 1]
```diff
@@ -555,11 +555,15 @@ SnippetPreview.prototype.renderOutput = function() {
  * Sets the classname of the meta field in the snippet, based on the rawData.snippetMeta
```
Comment (diedexx) [inline js/snippetPreview.js]: please update this comment

File: dist/yoast-seo.js [hunk 2]
```diff
@@ -3663,11 +3663,15 @@ SnippetPreview.prototype.renderOutput = function() {
  * Sets the classname of the meta field in the snippet, based on the rawData.snippetMeta
```
Comment (diedexx) [inline dist/yoast-seo.js]: please update this comment

**Comment by diedexx** [comment]
CR 1 done

**Comment by omarreiss** [comment]
CR :+1:

**Pull request merged**